<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/Set.png</normaloff>:/img/Set.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="check_OK">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_cancel">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>675</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTabWidget" name="Para">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>270</y>
     <width>770</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>Normal</string>
    </attribute>
    <widget class="QLineEdit" name="TimeFmt">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>15</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>yyyy/M/d h:m:s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Time Format</string>
     </property>
    </widget>
    <widget class="QLabel" name="Time_Lab">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>15</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1997/5/20 23:3:33</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TimePre">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>15</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>55</y>
       <width>230</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Threshold (min) of Home/Work Point</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>55</y>
       <width>190</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Day Time From/To (Hour only)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="paraH">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>55</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1440</number>
     </property>
     <property name="value">
      <number>240</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="paraW">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>55</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1440</number>
     </property>
     <property name="value">
      <number>240</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="paraD1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>615</x>
       <y>55</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>24</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="paraD2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>665</x>
       <y>55</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>24</number>
     </property>
     <property name="value">
      <number>22</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Process</string>
    </attribute>
    <widget class="QSpinBox" name="GrainSize">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>15</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="value">
      <number>1073741824</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="ProNo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>15</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>180</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Memery Mapping Grain Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>15</y>
       <width>30</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Byte</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>15</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Multi-processor Number</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="TempCheck">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>55</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>UpArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>Creat Temporary File</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TempPath">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>55</y>
       <width>300</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>55</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Buffer Size</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="BufferSize">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>55</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="value">
      <number>104857600</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>705</x>
       <y>55</y>
       <width>30</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Byte</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="FileButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>15</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>File...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cB0_usrID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QComboBox" name="cB1_time">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QComboBox" name="cB2_lng">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QComboBox" name="cB3_lat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>230</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QRadioButton" name="form_rad_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>55</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Comma(,)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="form_rad_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>55</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Tab(\t)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="form_rad_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>55</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="form_rad_3txt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>55</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="form_check_line1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>55</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>The First Line is also data.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>55</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="DataTable">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>95</y>
     <width>770</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UserID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lng</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>230</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_Reset">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>410</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
